app/upgrades.go

```diff
@@ -3,36 +3,19 @@ package app
 import (
 	"fmt"
 
-	"github.com/CosmWasm/wasmd/x/wasm"
-	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
 	store "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
 )
 
 // next upgrade name
-const upgradeName = "v3"
+const upgradeName = "v4"
 
 // RegisterUpgradeHandlers returns upgrade handlers
 func (app *App) RegisterUpgradeHandlers(cfg module.Configurator) {
 	app.UpgradeKeeper.SetUpgradeHandler(upgradeName, func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
-		newVM, err := app.mm.RunMigrations(ctx, cfg, vm)
-		if err != nil {
-			return newVM, err
-		}
-		// consensus params
-		// increase max gas as part of the upgrade to handle cosmwam
-		consensusParams := app.BaseApp.GetConsensusParams(ctx)
-		consensusParams.Block.MaxGas = 75_000_000 // 75M
-		app.BaseApp.StoreConsensusParams(ctx, consensusParams)
-
-		// wasm params
-		wasmParams := app.WasmKeeper.GetParams(ctx)
-		wasmParams.CodeUploadAccess = wasmtypes.AllowNobody
-		wasmParams.MaxWasmCodeSize = DefaultMaxWasmCodeSize
-		app.WasmKeeper.SetParams(ctx, wasmParams)
-		return newVM, err
+		return app.mm.RunMigrations(ctx, cfg, vm)
```

Comment:
Should we update the block per year while we're at it?

Reply:
aa great question but I'd rather see a different proposal